src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/DataSetCandidatesControllerTests.cs

```diff
@@ -258,11 +283,18 @@ public async Task ReleaseVersionDoesNotExist_Returns404()
             response.AssertNotFound();
         }
 
-        private async Task<HttpResponseMessage> GetDataSetCandidates(Guid releaseVersionId, HttpClient? client = null)
+        private async Task<HttpResponseMessage> GetDataSetCandidates(
+            Guid releaseVersionId,
+            ClaimsPrincipal? user = null
+        )
         {
-            var user = DataFixture.AuthenticatedUser().WithClaim(SecurityClaimTypes.AccessAllReleases.ToString());
+            var defaultUser = DataFixture
+                .AuthenticatedUser()
+                .WithClaim(SecurityClaimTypes.AccessAllReleases.ToString());
```

Comment:
I was just looking into whether this can be `OptimisedTestUsers.Authenticated` instead, and now I realise why we can't do this, when I spotted there's a warning here on `ToString`.

It looks like  `.WithClaim(nameof(SecurityClaimTypes.AccessAllReleases));` is preferred here.

Reply:
Can scoop them all up with regex replace:

`WithClaim\((.*)\.ToString\(\)\)`

to

`WithClaim(nameof($1))`